README.md

```diff
@@ -1,23 +1,22 @@
 # Virtual_ship_classroom
 Emma's work for the MSc student material
-Please contact me at e.e.daniels1@uu.nl with any questions. 
+Please contact me at e.e.daniels1@uu.nl with any questions.
 
-This is a python tool that will allow students to virtually sample the ocean as if the measurements were coming from an actual oceanographic mission. At the moment we mimic ADCP, CTD, and simple underwaydata measurements and allow surface drifters and argo float deployments. We might add gliders and meteorological data in the future. 
+This is a python tool that will allow students to virtually sample the ocean as if the measurements were coming from an actual oceanographic mission. At the moment we mimic ADCP, CTD, and simple underwaydata measurements and allow surface drifters and argo float deployments. We might add gliders and meteorological data in the future.
 
-### Requirements
-To use the material please create an environment called Parcels with the command
-`conda env create -f environment.yml
+### Installation
+Please follow the installation instructions detailed in the [contributing page](.github/CONTRIBUTING.md).
```

Comment:
Is it not confusing that installation is on the contributing page? Should there not (also) be an installation page? Most users wil want to install but not contribute

Reply:
I'm still to discuss with @ammedd how this tool will be exposed to students (i.e., how will this interface in the classroom). At the moment we just have a dev install (hence the intructions in the contributing file)

Reply:
I can get a conda installable package up and running (+pypi)